projects/gloo/api/v1/enterprise/options/ai/ai.proto

```diff
@@ -208,6 +208,66 @@ message UpstreamSpec {
     string model = 4;
   }
 
+  enum SigningAlgorithm {
+    // Use SigV4 for signing
+    AWS_SIGV4 = 0;
+
+    // Use SigV4A for signing
+    AWS_SIGV4A = 1;
+  }
+
+  // Settings for the Bedrock LLM provider
+  message Bedrock {
+    // The authorization config used to access authenticated AWS Bedrock services.
+    AwsCredentialProvider credential_provider = 1;
+    // Optional: Send requests to a custom host and port, such as to proxy the request,
+    // or to use a different backend that is API-compliant with the upstream version.
+    // Note: For AWS Bedrock, if custom_host is set, host_rewrite will be used to override the Host header before signing the request
+    CustomHost custom_host = 2;
+    // Optional: Sets the model-id name.
+    // If unset, the model name is taken from the request.
+    string model = 3;
+    // Required: region string.
+    //
+    // When signing_algorithm is set to ``AWS_SIGV4`` the region is a standard AWS `region <https://docs.aws.amazon.com/general/latest/gr/rande.html>`_ string for the service
+    // hosting the HTTP endpoint.
```

Comment:
Are the double ticks needed? also, what does the region wording seems different than the example, maybe rephrase.
```suggestion
    // When signing_algorithm is set to `AWS_SIGV4`, then the region is a string for the standard AWS region for the service that hosts the HTTP endpoint.
    // For more regions, see the AWS docs <https://docs.aws.amazon.com/general/latest/gr/rande.html>
```

Reply:
Actually should remove this sentence because the signing algorithm is not configurable currently!